sfge/src/main/java/com/salesforce/config/EnvUtil.java

```diff
@@ -21,6 +24,8 @@ public final class EnvUtil {
 
     @VisibleForTesting static final boolean DEFAULT_RULE_ENABLE_WARNING_VIOLATION = true;
     @VisibleForTesting static final boolean DEFAULT_IGNORE_PARSE_ERRORS = false;
+    @VisibleForTesting static final boolean DEFAULT_LOG_WARNINGS_ON_VERBOSE = false;
+    @VisibleForTesting static final int DEFAULT_PROGRESS_INCREMENTS_ON_VERBOSE = 50;
```

Comment:
Unnecessary, since progress logging should happen all the time instead of just on Verbose.